### ISSUE TITLE
ENH: add option to remove brackets from mapclassify legend

### ISSUE THREAD

**Issue by martinfleis** [geopandas/geopandas#1518]
Since 0.8.0, mapclassify indicates which value is part of the actual bin and which is not by `(` or `[` (as open and closed interval). I think it is a useful thing, but I was asked how to remove them, to keep only the numbers as it was in 0.7.0. We could add an option to `legend_kwds` which would use the old format. Something like `interval=True/False`.

New:
![Unknown](https://user-images.githubusercontent.com/36797143/87411515-bab3ea80-c5bf-11ea-8775-1f004f70afe5.png)

Old:
![Unknown-1](https://user-images.githubusercontent.com/36797143/87411721-05cdfd80-c5c0-11ea-97e7-d1a0806afd2f.png)

It is nothing crucial though, just a nice option to have.

**Comment by jorisvandenbossche** [comment]
Yes, I think that people will often want to remove those (personally, for a tweaked, to-be-published figure, I would remove it as well), so it would be good to have some flexibility regarding this.

**Comment by donlo** [comment]
Would like to take part in this one

https://github.com/geopandas/geopandas/blob/38fd095fa5a44219ce3b9b389b2109d67797e82c/geopandas/plotting.py#L696-L697

would it be a good workaround to add a check after the assignment above to remove the intervals?
```
    categories = binning.get_legend_classes(fmt)
    if interval:
        categories = [c.replace('(', '') ... for c in categories]
values = np.array(binning.yb)
```

**Comment by martinfleis** [comment]
@donlo feel free to open a PR with this. Implementation-wise, I would check for `interval` keyword in `legend_kwds`.

**Comment by StevenLi-DS** [comment]
This is intended? I think the old one makes much more sense.

**Comment by martinfleis** [comment]
@StevenLi-DS Yes, it is intended as it is more informative (you know which of the edge number is included where).

**Comment by StevenLi-DS** [comment]
@martinfleis and also I really wanna the legend circle to be squares or rectangles.
I don't think there is a way to control this in geopandas.
Is there any easy way to do this?

**Comment by martinfleis** [comment]
@StevenLi-DS agree. See #735 for a bit of a discussion on that.

**Comment by martinfleis** [comment]
Closed by #1605
